qa/common/util.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+
+util_sh_path = os.path.abspath("../common/util.sh")
+
+
+def check_test_results(test_log, expected_number_of_test):
+    try:
+        subprocess.run([
+            "bash", "-c", "source " + util_sh_path + "; check_test_results \"" +
+            str(test_log) + "\" \"" + str(expected_number_of_test) + "\""
+        ],
+                       check=True)
+    except subprocess.CalledProcessError:
+        print("\n***\n*** Test Output Verification Failed for " +
+              os.path.basename(test_log) + ".\n***")
+        return False
+    return True
+    """
```

Comment:
It not really supposed to be there, because the functions are to be converted to Python script, as you described above, in the next PR. Maybe I could just skip the next PR and convert with this one.

Reply:
Doing it in the next PR is ok too.

Reply:
This part will be moved into `util.sh`. This file will be removed.